### ISSUE TITLE
Object prototype may only be an Object or null

### ISSUE THREAD

**Issue by pmagnussen** [AgoraIO/React-Native-SDK#9]
I keep getting this issue when running the demo app:
'Object prototype may only be an Object or null'. It's in the `AgoraRtcEngineModule.js` file on line 3.

 I get the same error on a new react-native project. Any idea what it might be?

**Comment by cryptixcoder** [comment]
I am experiencing this same issue, really need to get this implemented...

**Comment by kunalqss** [comment]
Getting the same error running the demo app. Please help.

**Comment by rahmamajed** [comment]
same here any help about it ???

**Comment by cmder** [comment]
try to run `npm install` in the root directory of the project

**Comment by rahmamajed** [comment]
tried it nothing happened  
same error

**Comment by rahmamajed** [comment]
> install

tried it nothing happened
same error

**Comment by cmder** [comment]
Have u add the Agora SDK to the specific directory and build the android project in Android Studio properly?

**Comment by rahmamajed** [comment]
> Have u add the Agora SDK to the specific directory and build the android project in Android Studio properly?

yes i take the demo files and i run it and it give me this error.

**Comment by cmder** [comment]
u mean u run the project in Android Studio ? u may not run it like that, u should run with `reactive-native run-android`

**Comment by rahmamajed** [comment]
> u mean u run the project in Android Studio ? u may not run it like that, u should run with `reactive-native run-android`

i do it on react native not in the android 
i was talking about the demo react project when i run it it give me the same result 
also i try to make new project and follow all the steps on the document same error

**Comment by Rohan5696** [comment]
is issue resolved or not??

**Comment by rahmamajed** [comment]
no still

**Comment by sibigit** [comment]
still the same error any solution?

**Comment by plutoless** [comment]
For all users who asked the question in this issue thread:
Today i double checked from my side to verify if everything works fine, here's what i do.
1. clone the repo
2. do npm install
3. run react-native run-android

and the program can successfully start w/o this error.
I'm using android studio 3.3 + jdk 1.8

If you are doing above actions but still suffer issues. Please describe your detail step here with screenshots.

**Comment by freddneos** [comment]
Same issue here ...

Android : Object prototype may only be an Object or null
Line : let AgoraRtcEngine = Object.create(NativeModules.AgoraRtcEngineModule);

I follow the instructions and faced it.

**Comment by rrscodes** [comment]
Same Issue here as well.

**Comment by plutoless** [comment]
@roshansingh21 this repository has been deprecated. please use this instead https://github.com/AgoraIO-Community/Agora-RN-Quickstart
